MAINTAINERS

```diff
@@ -20,7 +20,10 @@ Jonas Östanbäck <jonas.ostanback@gmail.com> (@cez81)
 David Schneiderbauer <dschneiderbauer@gmail.com> (@daviian)
 Peter Žeby <morlinest@gmail.com> (@morlinest)
 Matti Ranta <matti@mdranta.net> (@techknowlogick)
-Michael Lustfield <mtecknology@debian.org> (@MTecknology)
```

Comment:
Some maintainers are removed in this PR

Reply:
Related go-gitea/blog#83 (comment), go-gitea/gitea#5599 and https://github.com/go-gitea/gitea/pull/4979